app/src/main/java/com/example/mvvmcleanarchitecture/features/characters/domain/model/Character.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.mvvmcleanarchitecture.features.characters.domain.model
+
+data class Character(
+    val id: Int,
+    val created: String,
+    val episode: List<String>,
+    val gender: String,
+    val image: String,
+    val name: String,
+    val origin: Origin,
+    val species: String,
+    val location: Origin,
```

Comment:
Tutaj w kontekście nazewnictwa wydaje mi się, że nie jest do końca spójnie. Prędzej jakiś obiekt o nazwie `Location` może modelować miejsce pochodzenia oraz miejsce, w którym dana osoba się aktualnie znajduje. W drugą stronę jest to trochę mało intuicyjne, bo typ `Origin` w zasadzie mówi tylko o pochodzeniu i używanie go w kontekście aktualnego miejsca pobytu nie jest do końca trafione.

Reply:
To oczywiście tyczy się modelu `Character` na każdej z warstw.